Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3933,6 +3933,19 @@ InspectorStartsAttached:
     WebKit:
       default: true
 
+InspectorStoreCertificateInfo:
+  type: bool
+  status: embedder
+  defaultValue:
+    WebKitLegacy:
+      default: false
+    WebKit:
+      "PLATFORM(COCOA)": true
```

Comment:
`http/tests/inspector/network/getSerializedCertificate.html` failed for win-tests EWS.
Do we need `"PLATFORM(WIN)": true` line here?